README.md

```diff
@@ -12,9 +12,10 @@
       src="https://cdn.rawgit.com/webpack/media/e7485eb2/logo/icon.svg">
   </a>
   <h1>UglifyJS Webpack Plugin</h1>
-	<p>This plugin uses <a href="https://github.com/mishoo/UglifyJS2/tree/v2.x">UglifyJS v2</a> to minify your JavaScript.<p>
```

Comment:
Yeah idk I'm not exactly sure why the indentation is causing issues but if that's the case, maybe the markdown parsing process or readme parsing process should be tweaked for now. I agree with Juho that it would be really nice to agree on some standards at __webpack-defaults__ because I feel like a lot is falling on our markdown parsing process in the docs repo. 

I'm not too familiar with __markdown-magic__ but I feel like one solution would be to just support the exact same markdown extensions that github does (which is basically what we're hackily trying to do right now). However, they haven't open-sourced their parser as far as I know although I do think I've read that they use `marked` and customize it (as do we). I was hoping to use [marksy](https://github.com/cerebral/marksy) to clean things up as I think it'll allow cleaner customizations but that also uses `marked` under the hood and doesn't support everything github does 😢 .

TLDR; yeah I think we'd accept a hack for now on the docs repo (either in the markdown or readme processing) but, as Juho said, we really need to come to some agreement as to what should be supported and how because the problem will just keep getting worse.